client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -165,10 +168,12 @@ export default React.createClass( {
 			return <PluginInstallButton { ...this.props } />;
 		}
 
+		const { isTransferring } = this.props;
+
 		if ( this.props.selectedSite && ! this.props.selectedSite.jetpack ) {
 			return (
 				<WpcomPluginInstallButton
-					disabled={ ! this.hasBusinessPlan() }
+					disabled={ ! this.hasBusinessPlan() || isTransferring }
```

Comment:
it shouldn't matter here but just be aware that the value of `isTransferring` here could also be `null`